app/app/email-templates/src/new_admin.mjml

```diff
@@ -0,0 +1,14 @@
+<mjml>
```

Comment:
what's mjml???

Reply:
mjml.io, I copied and edited an existing template and generated the html using their website